.github/workflows/dependent-issues.yml

```diff
@@ -0,0 +1,36 @@
+---
+name: Dependency Checks
+
+on:
+  issues:
+    types:
+      - opened
+      - reopened
+  pull_request_target:
+    types:
+      - opened
+      - reopened
+  pull_request:
+    types:
+      - opened
+      - reopened
+  schedule:
+    - cron: '0 0 * * *' # check daily
+
+jobs:
+  check:
+    name: PR Dependency
```

Comment:
```suggestion
    name: Update PR Dependency Labeling
```

Maybe?

Reply:
It is not just updating the labeling. Its checking for dependency among PRs

Reply:
But when it finds dependencies, doesn't it maintain a list in comments (basically labeling)?

Reply:
Not exactly.Initially we add the comment saying that PR1 is dependent on PR2. So this dependency job on PR1 will keep on failing until PR2 is merged. Once PR2 is merged, this job will notify that PR2 is merged and will pass for PR1. So we can then safely merge PR1 as well

Reply:
Right, so "labeling" is not a great name. How about something like "Check Dependencies", paired with the PR-naming context of the Workflow?